### DISCUSSION TITLE
Сброс времени полета

### DISCUSSION THREAD

**Discussion by Flackon88** [alexey-gamov/opentx-quad-telemetry#2] [Q&A] [answered]
В скрипте предусмотрен сброс времени полета?

Установил на ТХ-12, через `Reset telemetry` время полета не сбрасывается.

_Протокол FrSky D-16_

**Comment by alexey-gamov** [comment, marked as answer]
Долго обдумывал сброс времени: вариантов было несколько, но все они так или иначе приводят к вопросам в использовании.

Сейчас реализовано по-простому, так же как и в системе таймеров на OpenTX.

Для сброса нужно зажать: <kbd>MENU</kbd> - <kbd>Reset timer</kbd>

**Comment by alexey-gamov** [comment]
Сейчас если в модели таймер не настроен - то работает по принципу расчёта времени нахождения квада в АРМе.

Так же можно использовать как таймер обратного отсчета: если в модели установить таймеру произвольное значение _(прим. 02:00)_ то полётное время пойдет на убывание. После прохождения 0 точки начнет мигать и показывать время сверх установленного.

**Comment by Flackon88** [comment]
Установил `Timer1` на свитч арма, в `Persist.` указал `Flight`
Работает при Арме и так же сбрасывается через <kbd>MENU</kbd> - <kbd>Reset Flight</kbd>
